src/main/java/edu/unc/lib/boxc/migration/cdm/status/SourceFilesSummaryService.java

```diff
@@ -91,6 +92,96 @@ public int totalFilesInProject() {
         return getQueryService().countIndexedFileObjects();
     }
 
+    /**
+     * @return load list of all CSV records
+     */
+    public List<CSVRecord> loadAllEntries(Path mappingPath) {
+        List<CSVRecord> allEntries = new ArrayList<>();
+
+        try (
+                Reader reader = Files.newBufferedReader(mappingPath);
+                CSVParser csvParser = new CSVParser(reader, CSVFormat.DEFAULT
+                        .withFirstRecordAsHeader()
+                        .withHeader(SourceFilesInfo.CSV_HEADERS)
+                        .withTrim());
+        ) {
```

Comment:
I believe you can use `SourceFileService.openMappingsParser()` here to open the parser